protoc_plugin/test/json_test.dart

```diff
@@ -161,17 +162,30 @@ void main() {
   test('testUnknownEnumValueInRepeatedField', () {
     // repeated NestedEnum repeated_nested_enum = 51;
     var message = TestAllTypes.fromJson('{"51": [4]}');
-    // 4 is an unknown value.
-    expect(message.repeatedNestedEnum, isEmpty);
+    // 4 is an unknown value, which should default to the enum's default value.
+    expect(message.repeatedNestedEnum, equals([TestAllTypes_NestedEnum.FOO]));
+
+    // 1 (FOO) and 2 (BAR) are known values. All unknowns should fill in to
+    // the default enum value (FOO).
 
     // 1 (FOO) and 2 (BAR) are known values.
```

Comment:
Ah, this comment is actually meant to be replaced with the longer above comment.

Reply:
done